tox.ini

```diff
@@ -1,6 +1,9 @@
 [tox]
 envlist = py27, py34, py35
 
+[testenv:docs]
+commands = python setup.py build_sphinx
```

Comment:
This is here because if any PR checks in stuff that breaks docs generation it should be found quickly. Yes, i can add a follow up PR to do an addition script in the scripts folder later after this merges